docs/en/sql-reference/statements/select/order-by.md

```diff
@@ -221,3 +221,70 @@ returns
 │ 1970-03-12 │ 1970-01-08 │ original │
 └────────────┴────────────┴──────────┘
 ```
+
+## OFFSET FETCH Clause {#offset-fetch}
+
+`OFFSET` and `FETCH` allow you to retrieve just a portion of the rows that are generated by the rest of the query.
+
+``` sql
+OFFSET offset_row_count {ROW | ROWS} FETCH {FIRST | NEXT} fetch_row_count {ROW | ROWS} {ONLY | WITH TIES}
+```
+
+The `FETCH` is an alternative to the [LIMIT] (../../../sql-reference/statements/select/limit.md) clause and retrieves rows from a query `SELECT` type.
```

Comment:
1. Link to LIMIT dosn't work.
2. FETCH is not a query, it doesn't retrieve rows, it specifies the maximum number of rows that can be in the result of a query.
3. You are describing the SELECT...ORDER BY query. No need to emphasize the type.